src/MatLang.jl

```diff
@@ -1,23 +1,167 @@
 module MatLang
 using Requires
+using LinearAlgebra
 
-export clcM
+# Main Functions
+export clcM, zerosM, onesM, randM, eyeM, diagM
+
+# Julia functions with M suffix
+# export
+
+################################################################
+################################################################
```

Comment:
To be honest, it's still a little wired to see all these `######...` Most Julia packages don't really have this. We simply split functions into different files and use `include` to hold them together.

Reply:
That's good visual separation, at least for my eyes. 😄 I will create separate files as the package grows. Like for one file for strings and another for arrays.

Reply:
Anyway, it doesn't matter much here; I just tell you my feelings.